driver/fdo.go

```diff
@@ -11,18 +11,54 @@ import (
 	"fyne.io/fyne"
 )
 
-var iconExtensions = []string{".png", ".svg"}
+var (
+	iconExtensions = []string{".png", ".svg"}
+	iconTheme      string
```

Comment:
Having these global variables in the driver package seems strange (I'll reference them below)